kunta-api-server/src/main/java/fi/otavanopisto/kuntaapi/server/rest/OrganizationsApiImpl.java

```diff
@@ -122,8 +135,19 @@ public Response deleteService(String organizationId, String serviceId) {
   }
   
   @Override
-  public Response listServiceElectornicChannels(String organizationId, String serviceId) {
-    return createNotImplemented("Not implemented");
+  public Response listServiceElectornicChannels(String organizationIdParam, String serviceIdParam) {
+    OrganizationId organizationId  = new OrganizationId(KuntaApiConsts.IDENTIFIER_NAME, organizationIdParam);
+    ServiceId serviceId = new ServiceId(KuntaApiConsts.IDENTIFIER_NAME, serviceIdParam);
+    
+    // TODO: Merge provider results
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)